regression/cbmc-java/slice1/test.desc

```diff
@@ -0,0 +1,6 @@
+KNOWNBUG
```

Comment:
It seems rather impossible to infer what the problem here is from the commit message. Please either make the commit message more verbose or add comments to test.desc.